.env-ui.sample

```diff
@@ -0,0 +1,2 @@
+ISSUER_UI_BASIC_USERNAME=user
+ISSUER_UI_BASIC_PASSWORD=password
```

Comment:
To discuss: move `ISSUER_API_UI_ISSUER_NAME` & `ISSUER_API_UI_ISSUER_LOGO=` to here (as `ISSUER_UI_ISSUER_NAME` & `ISSUER_UI_ISSUER_LOGO=`), since the API doesn't need to know about them (consuming the name and logo is only relevant for the UI). This would simplify the API payload too.

Reply:
Added as an [issue](https://polygonid.atlassian.net/browse/PID-344).